src/renderers/contentful/renderContentType.ts

```diff
@@ -67,7 +68,7 @@ function renderSys(sys: Sys) {
       locale: string;
       contentType: {
         sys: {
-          id: '${sys.id}';
+          id: '${escapeSingleQuotes(sys.id)}';
```

Comment:
AFAIK `sys.contentType.sys.id` is the api identifier for the content model, which can only have letters and numbers - so escaping might not be necessary here:

![image](https://user-images.githubusercontent.com/251391/133575872-a70ef188-b544-4529-bd6a-fa1c15ddaa63.png)

I might be wrong though 😅 Let me know what you think

Reply:
Yea this doesnt need to be escaped I kinda just went through and find and replaced, lemme remove it from this.

Reply:
Perfect! Looks great! Thanks @Jaden-Giordano!